python/nav/web/maintenance/urls.py

```diff
@@ -25,8 +25,8 @@
 urlpatterns = [
     path('', views.redirect_to_calendar),
     # XXX: WTF?
-    re_path(r'^calendar/$', views.calendar, name='maintenance'),
-    re_path(r'^calendar/$', views.calendar, name='maintenance-calendar'),
+    path('calendar/', views.calendar, name='maintenance'),
```

Comment:
Should probably add a follow-up task by removing the `name='maintenance'` path. It seems to only be referenced in the `redirect_to_calendar` view, which should be modified to just use `'maintenance-calendar'` instead.